google/cloud/aiplatform/datasets/tabular_dataset.py

```diff
@@ -73,20 +73,28 @@ def column_names(self) -> List[str]:
                 gcs_source_uris.sort()
 
                 # Get the first file in sorted list
-                return TabularDataset._retrieve_gcs_source_columns(
-                    self.project, gcs_source_uris[0]
+                return self._retrieve_gcs_source_columns(
```

Comment:
Is there any use case where the user would want to pass in different credentials when calling `dataset.column_names`?

If not, this looks fine.